### ISSUE TITLE
Entry abstract not included

### ISSUE THREAD

**Issue by arituhkala** [sciunto-org/python-bibtexparser#80]
Hi,

Thanks for nice parser. Seems like abstract is not included in the entry dictionary even though documentation says so. I've tried many different ways.

bibtex:

```
@article{Ninlawan20151732,
title = "Factors Which Affect Teachers’ Professional Development in Teaching Innovation and Educational Technology in the 21st Century under the Bureau of Special Education, Office of the Basic Education Commission ",
journal = "Procedia - Social and Behavioral Sciences ",
volume = "197",
number = "",
pages = "1732 - 1735",
year = "2015",
note = "7th World Conference on Educational Sciences ",
issn = "1877-0428",
doi = "http://dx.doi.org/10.1016/j.sbspro.2015.07.228",
url = "http://www.sciencedirect.com/science/article/pii/S1877042815042299",
author = "Ganratchakan Ninlawan",
keywords = "the 21st Century",
keywords = "Teachers’ Professional ;Special Education ; ",
abstract = "Abstract The study aimed to investigate factors which affect teachers’ professional development in teaching innovation and educational technology in the 21st century under the Bureau of Special Education, Office of Basic Education. There were 400 participants. The statistical tool used in the study was Multiple Regression Analysis. The independent variables were entered in the stepwise method. The study found that there were positive correlations between teachers’ professional development and classroom management in the 21st century, concerning creative and innovative skills, communication, information, and media awareness, and computer literacy and information technology. The correlation coefficients were at .295, .349, and .408 respectively. "
}
```

Result:

```
{'year': '2015', 'volume': '197', 'title': 'Factors Which Affect Teachers’ Professional Development in Teaching Innovation and Educational Technology in the 21st Century under the Bureau of Special Education, Office of the Basic Education Commission', 'doi': 'http://dx.doi.org/10.1016/j.sbspro.2015.07.228', 'note': '7th World Conference on Educational Sciences', 'pages': '1732 - 1735', 'keyword': 'Teachers’ Professional ;Special Education ;', 'number': '', 'journal': 'Procedia - Social and Behavioral Sciences', 'issn': '1877-0428', 'ID': 'Ninlawan20151732', 'author': 'Ganratchakan Ninlawan', 'link': 'http://www.sciencedirect.com/science/article/pii/S1877042815042299', 'ENTRYTYPE': 'article'}
```

**Comment by sciunto** [comment]
That's a good point. It's definitely a mistake and we should do it as the documentation says.
I'll fix this asap. Sorry for being slow.

**Comment by sciunto** [comment]
Will be fixed by #64. Thanks for reporting.
